edx/edx-enterprise
# [ENT-491] Remove SSO consent features; verify fully-populated consent URL generation

**Description:** This pull request removes the ability to request account-level consent via a linked SSO provider. Consent is now a separate operation.

**JIRA:** [ENT-491](https://openedx.atlassian.net/browse/ENT-491)

**Dependencies:** edx/edx-platform#15624

**Merge deadline:** 31 July 2017

**Installation instructions:** N/A

**Testing instructions:**

1. Verify that you can no longer set an EnterpriseCustomer to have "optional" or "at login" consent requirements.

**Merge checklist:**

  **DO NOT** just add dependencies to `requirements/*.txt` files.

**Post merge:**
